src/App.vue

```diff
@@ -1,40 +1,52 @@
 <template lang="pug">
 v-app.app
-  v-dialog(v-model="loggedOut", width="50%", persistent)
-    girder-auth(
-        :register="true",
-        :oauth="true",
-        :forgot-password-url="forgotPasswordUrl")
-  v-dialog(v-model="uploader", width="70%")
-    girder-upload(v-if="uploader",
-        :dest="uploadDest",
-        multiple="multiple",
-        @done="$refs.girderBrowser.refresh()")
-  girder-data-browser(ref="girderBrowser",
-      v-if="!loggedOut && location",
-      :location.sync="location",
-      @click:newitem="uploader = true")
+  v-container
+    h4.display-1.pb-3 Girder Vue Demo
+    v-dialog(v-model="loggedOut", full-width, max-width="600px")
+      girder-auth(
+          :register="true",
+          :oauth="true",
+          :forgot-password-url="forgotPasswordUrl")
+    v-dialog(v-model="uploader", full-width, max-width="800px")
+      girder-upload(v-if="uploader",
+          :dest="uploadDest",
+          multiple="multiple",
+          @done="$refs.girderBrowser.refresh(); uploader = false;")
+    v-dialog(v-model="newFolder", full-width, max-width="800px")
+      girder-upsert-folder(v-if="newFolder",
```

Comment:
What is the reason we would use the v-if for the content of the dialog and the sibling dialogs? They actually make the dialog disappear animation not working correctly.